config/routes.rb

```diff
@@ -123,7 +123,9 @@
     end
   end
 
-  resources :usage, only: [:index]
+  resources :usage, only: [:index] do
```

Comment:
It looks like you're just defining an index action here. Can we move this into its own controller (`UsageDownloadsControlller#index`, or into the `UsageController#index` action?

Also, remember to use plural names for Rails resources.